### DISCUSSION TITLE
Support of other Question Formats

### DISCUSSION THREAD

**Discussion by CrazyWolf13** [openmultiplechoice/openmultiplechoice#830] [Q&A] [answered]
Hi
our app is really really nice and there is very little other apps in this category, barely and maintained app that could compare against Quizlet, are there any plans to further integrate learning or answering modes ?

Like for example language 1 is displayed, you have to enter language2 it's then correct or wrong, at the end a counter and option to save failed cards and / or redo them.

Thank you!

Greetings from Switzerland!

**Comment by schu** [comment, marked as answer]
Thank you for your interest and question!

For the foreseeable future and towards v1.0, adding more question types isn't planned and our focus is to stabilize and improve the current feautures and to add more quality of life, UI and UX improvements around those.

But we are happy to receive and gather feedback and ideas nonetheless and there is a chance that we add more question types and session modes in the future.
